testing/jormungandr-integration-tests/src/jormungandr/explorer/explorer_sanity.rs

```diff
@@ -61,7 +61,8 @@ pub fn explorer_sanity_test() {
     let (jormungandr, initial_stake_pools) =
         startup::start_stake_pool(&[faucet.clone()], &[], &mut config).unwrap();
 
-    let explorer_process = jormungandr.explorer();
+    let params = ExplorerParams::new("70".to_string(), None, None);
```

Comment:
magic number / string, plus isn't it already a string?

Reply:
It is a &str and the function requires a String, so conversion is needed:) I'll fix the magic